### ISSUE TITLE
invalid homepage url -- potential validation error?

### ISSUE THREAD

**Issue by rnjudge** [spdx/tools-python#817]
I am using the python tools library to validate an SBOM. I get an error about an invalid homepage even though it appears valid to me:

```
[rose@fedora ]$ pyspdxtools -i spdx.json 
ERROR:root:The document is invalid. The following issues have been found:
homepage must be a valid URL, but is:  https://github.com/KhronosGroup/Vulkan-headers
```

This seems like a valid link to me.... Thanks in advance for any insight.

**Comment by rnjudge** [comment]
Thanks for catching that @billie-alsup!

**Comment by billie-alsup** [comment]
What should the behavior be in this case?

- Should the parser strip leading and trailing spaces on a url when loading into a Document/Package/etc.?
- Should the validator ignore leading/trailing spaces on a url?
- Should it be treated as an error?

**Comment by goneall** [comment]
I don't have a strong opinion on validation of the homepage URL, but following are a couple of thoughts / reccomendations:
- In tag/value, we need to trim since the format is not very precise for character handling, for other formats (like JSON), we may not want to do the trimming for validation warnings.
- The parser should not error out (it sounds like it isn't) due to leading characters, but generating a warning in terms of a validation message is fine.

**Comment by armintaenzertng** [comment]
I agree with Gary. Sounds to me like the issue can be closed. Please reopen if you disagree or find any other unwanted behavior.
